lms/djangoapps/support/docs/decisions/0001-sso-history-in-support.rst

```diff
@@ -0,0 +1,34 @@
+1. Registering SSO History Model in Support
+================================================
+
+Status
+------
+
+Accepted
+
+Context
+-------
+
+SSO History was one of the feature requested by support that provides the 
+historical data of any particular SSO record (based off UserSocialAuth)
+in frontend-app-support-tools via support API.
+
+Decision
+--------
+
+We have introduced the simple_django_history registeration for 
```

Comment:
Instead of We, let's use a third-person perspective in the writing. The reader will not know who "we" are.